draft-ietf-quic-transport.md

```diff
@@ -3425,6 +3433,10 @@ immediately or when a delayed ack timer expires. The delayed ack timer MUST
 NOT delay an ACK for longer than an RTT, which ensures an ACK frame is sent
 at least once per RTT if new packets needing acknowledgement were received.
 
+If the max_bytes_before_ack transport parameter has been received, the
+receiver SHOULD send an ACK frame immediately once that many octets of packets
+containing frames besides ACK or PADDING have been received.
```

Comment:
What is the logic you are looking for?  Count the total size of received packets, unless they are ACK-only, then send an ACK when count>max_bytes_before_ack?

I would reframe this slightly in that case.  "The max_bytes_before_ack transport parameter can be used to request more frequent acknowledgments.  After sending a packet containing an ACK frame, a receiver counts the size of all packets it receives, omitting packets that only contain ACK or PADDING frames.  Once this count exceeds the value of the max_bytes_before_ack transport parameter it has received from a peer, the receiver MUST send an ACK frame immediately."

Reply:
IoT devices may receive data at a lower power cost than transmitting. Firing up the transmit radio could be costly. For these applications it makes more sense to transmit ACK's at timed intervals. They might skip an interval if to few bytes have been received.

Reply:
Yeah, I don't think we want MUST, but I'll add a caveat that if you don't do what the peer requests, you risk reduced performance.